themes/default/content/docs/concepts/inputs-outputs/apply.md

```diff
@@ -0,0 +1,822 @@
+---
+title_tag: "Accessing Single Outputs with Apply | Inputs and Outputs"
+meta_desc: "Learn how to access a single output value using the apply method in Pulumi."
+title: Accessing single outputs with Apply
+h1: Accessing single outputs with Apply
+meta_image: /images/docs/meta-images/docs-meta.png
+menu:
+  concepts:
+    weight: 2
+    parent: inputs-outputs
+---
+
+## Overview
+
+[Outputs](/docs/concepts/inputs-outputs/#outputs) are asynchronous, meaning their actual plain values are not immediately available. Their values will only become available once the resource has finished provisioning. The asynchronous nature of Outputs is also why, when certain operations such as [`pulumi preview`](/docs/cli/commands/pulumi_preview/) runs, the outputs for a new resource do not yet have any possible values. As such, there are limitations on the ways in which you can retrieve and interact with these values.
```

Comment:
I wonder if adding both the reason + benefits of Pulumi would be helpful? (to folks new to pulumi at least but we could save it for a tutorial)  

The reason outputs are asynchronous is because provisioning resources is an asynchronous operation. It takes time for a cloud provider and Pulumi optimizes the process by executing operations in parallel, rather than sequentially.

Reply:
I'll include this explanation on the landing page